projects/arlas-components/src/lib/components/results/result-list/result-list.component.ts

```diff
@@ -454,7 +454,13 @@ export class ResultListComponent implements OnInit, DoCheck, OnChanges, AfterVie
    * @Output : Angular
    * @description Emits when result list is updated.
    */
-  @Output()  public onResultListUpdate: EventEmitter<boolean> = new EventEmitter<boolean>();
+  @Output() public onResultListUpdate: EventEmitter<boolean> = new EventEmitter<boolean>();
+
+  /**
+   * @Output : Angular
+   * @description Emits when the list is ready.
+   */
+  @Output() public onListLoaded = new EventEmitter<boolean>();
```

Comment:
Do you think we can use the output onResultListUpdate ? without adding a new output ? i see that they will not be fired at the same moment, I am just wondering

Reply:
I wanted to have an output specifically for that, kind of like the map with the onMapLoaded, to be sure that the component is well initialized. The onResultListUpdate is there for when the data is updated on every DoCheck, I am not sure whether there could be data arriving before the list is initialized and whether we could use the first firing of the event as an event. For clarity purposes, I think it is better to keep both separated, so that people would know which one to use for each use case.